framework/src/st_things/things_stack/src/common/framework/things_data_manager.c

```diff
@@ -660,18 +660,22 @@ wifi_manager_softap_config_s *dm_get_softap_wifi_config(void)
 {
 	st_device_s *device = (st_device_s *) hashmap_get(g_device_hmap, (unsigned long)(0));
 	int ssid_type = (is_artik == true ? 1 : 0);
-	unsigned char mac_id[16] = {0, };
+	unsigned char mac_id[16] = { 0, };
+	char ssid_device_name[17];
```

Comment:
Can you explain how did you arrive at this number 17 ?

Reply:
@pradeepns85 Now, 'ap_config.ssid' supports only 32 bytes for SSID becuase of h/w issue. 
And 'Samsung Connect' app needs 15 fixed bytes in SSID to distinguish each devices. (ex. _E1AAA100108BA8)
Therefore, the length of device name should be 16 (32 = 16(device name) + 15(fixed size) + 1 null char).